### PR TITLE
Make sure we use "set -o errexit" in all our script files

### PR DESCRIPTION
**Pull request by duglin**
Closes: #164

Signed-off-by: Doug Davis <dug@us.ibm.com>

### CODE REVIEW

**Comment by MHBauer** [comment]
`set -u` -> `set -o nounset` to be consistent. I prefer the longer more explicit names.

**Comment by duglin** [comment]
I thought about that too but I decided to limit the impact on existing scripts since we didn't formally say we'd require the longer version. But at least the for files I've touched I'll tweak 'em, as you said, for consistency.

File: build/verifyScripts.sh [hunk 1]
```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
```
Comment (pmorie) [inline build/verifyScripts.sh]: can we call this `verify-errexit.sh`, and push into repo-infra as a follow-up?

File: build/verifyScripts.sh [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+# Copyright 2015 The Kubernetes Authors.
```
Comment (pmorie) [inline build/verifyScripts.sh]: 2017

File: build/verifyScripts.sh (hunk 1 above)
Comment (duglin) [inline build/verifyScripts.sh]: done

File: build/verifyScripts.sh (hunk 2 above)
Comment (duglin) [inline build/verifyScripts.sh]: fixed

**Comment by duglin** [comment]
@pmorie edits done

**Pull request merged**